src/components/Modal/Modal.tsx

```diff
@@ -74,10 +74,14 @@ export function Modal(props: ModalProps) {
 
   const [hasScroll, setHasScroll] = useState(forceScrolling ?? false);
 
-  useResizeObserver(modalBodyRef, ({ target }) => {
-    if (forceScrolling === undefined && !isFixedHeight) {
-      setHasScroll(target.scrollHeight > target.clientHeight);
-    }
+  useResizeObserver({
+    ref: modalBodyRef,
+    onResize: useCallback(() => {
+      const target = modalBodyRef.current!;
+      if (forceScrolling === undefined && !isFixedHeight) {
+        setHasScroll(target.scrollHeight > target.clientHeight);
+      }
+    }, []),
```

Comment:
**thought/question:** I've never seen an inline `useCallback` like this. Was this something we could do in React 16?